From d739811038c54081ea4039a939af8f320b31378b Mon Sep 17 00:00:00 2001
From: Tor Lillqvist <tml@collabora.com>
Date: Tue, 29 Mar 2016 14:02:02 +0300
Subject: The OpenCL shared library is called libOpenCL.so.1 on Linux

libOpenCL.so is just the symbolic link to it, typically from
a development package. (On Fedora, from ocl-icd-devel.)

(Sure, libOpenCL.so.1 is also a symbolic link on Fedora, to
libOpenCL.so.1.0.0. But anyway, libOpenCL.so.1 is not from a
development package, but from ocl-icd.)

Change-Id: I18ca2e52e025001f709b1b57e12cd01d5b52d08e

diff --git a/opencl/source/openclwrapper.cxx b/opencl/source/openclwrapper.cxx
index 7bbcb2c..74a925b 100644
--- a/opencl/source/openclwrapper.cxx
+++ b/opencl/source/openclwrapper.cxx
@@ -38,7 +38,7 @@
 #elif defined(MACOSX)
 #define OPENCL_DLL_NAME nullptr
 #else
-#define OPENCL_DLL_NAME "libOpenCL.so"
+#define OPENCL_DLL_NAME "libOpenCL.so.1"
 #endif
 
 #define DEVICE_NAME_LENGTH 1024
-- 
cgit v0.10.2

